docker/build/releases-page.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install git python3 python3-pip -y && \
```

Comment:
Oh, this is fun. pip3 doesn't like global installations and will error. We should:
- add `python3-boto3`
- remove the pip3 install line on line5
- add the awscli install code we added to other files, i.e.:

```
RUN apt-get install -y curl unzip
RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
RUN unzip awscliv2.zip
RUN ./aws/install --bin-dir /usr/local/bin --install-dir /usr/local/aws-cli --update
```

Reply:
Breaks because of ubuntu upgrade.